src/components/EntityInfo/EntityMenu.vue

```diff
@@ -250,6 +250,10 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
     return !!GetFeatureFlag('enable-agm-location-chg')
   }
 
+  get isDCRoute (): boolean {
+    return this.$route.matched.some(route => route.name === DCRoutes.DIGITAL_CREDENTIALS)
```

Comment:
This isn't working and is causing a unit test error.

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/8b22994a-978b-4a6c-a80d-fcb0e36e0304)

PS You can do `npm run test:unit EntityMenu` locally.

Reply:
Right. I meant to fix that before pushing the PR

Reply:
Updated the tests and logic for hiding the menu option

Reply:
No test error but I still see the entity menu.

Reply:
Even after the last commit? https://github.com/bcgov/business-filings-ui/pull/562/commits/678323ca39e1d31f261285e438055593dd332568

Reply:
Yes. I built the preview site after your commits.

Trying again, just in case.

Reply:
Yes, entity menu still shows.

Reply:
Just re deployed. The menu should be gone now

Reply:
Please try it. I still see the menu.

Reply:
https://www.loom.com/share/9e1b651d4f084c5a8300c257112a8330?sid=eec62375-a9b7-41d7-a277-d14ab72fb3e9

Is it possible you have a cached version of the of app? Could you try clearing the cache and reloading the app?

Reply:
Just to confirm, the ?More Actions" menu should be available. Just the Digital Business Card option should be hidden.

Reply:
What your describe is how it is, but I think the entire menu should be hidden.

Resolving this conversation as there's another one related to this.